script/generate-metadata-for-builder.js

```diff
@@ -0,0 +1,209 @@
+'use strict';
+
+const CSON = require('season');
+const fs = require('fs-plus');
+const normalizePackageData = require('normalize-package-data');
+const path = require('path');
+const semver = require('semver');
+
+module.exports = function(packageJSON) {
```

Comment:
```suggestion
module.exports = function generateMetaDataForBuilder(packageJSON) {
```

I'd also rather not use "default" exports.